lusidtools/cocoon/seed_sample_data.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+from lusidtools.cocoon.cocoon import load_from_data_frame
+from lusidtools.cocoon.utilities import load_json_file
+import logging
+
+logger = logging.getLogger()
+
+mappings = dict(load_json_file("config/seed_sample_data.json"))
+
+def seed_sample_data(api_factory, file_types, scope: str, transaction_source, mappings: dict = mappings):
+
+    """
+    Description:
+    This function allows users to seed their LUSID environment with some core data (e.g. instruments,
+    portfolios, and transaction) from one file.
+
+    :param lusid.utilities.ApiClientFactory api_factory: The api factory to use.
+    :param list[str] file_types: A list of the file_types for upload.
+    :param str scope: The scope of the transaction portfolio.
+    :param str transaction_source: a CSV file of data.
+    :param dict mappings: a file containing mapping of CSV or DataFrame headers to LUSID headers.
+    :return: various LUSID request responses.
+    """
+
+    data_frame = pd.read_csv(transaction_source)
+
+    def dynamic_load_from_data_frame(file_type):
```

Comment:
what does the `dynamic` bit refer to?

Reply:
Changed the name to `generic_load_from_data_frame`.
I am trying to point out that the function is domain agnostic. 
Technically, this function is not needed. I could just loop over the standard `load_from_data_frame` function but I think the code looks cleaner this way?